Build fails due to bindgen being unable to find Windows include directories

```
  --- stderr
  cargo:rustc-link-lib=static=lzvn
  cargo:rustc-link-search=native=/io/target/x86_64-pc-windows-msvc/release/build/sepsplit-rs-hash/out
  cargo:rerun-if-changed=wrapper.h

  --- stderr
  ./src/ext/lzvn/FastCompression.h:8:10: fatal error: 'stdlib.h' file not found
  thread 'main' panicked at 'Unable to generate bindings: ClangDiagnostic("./src/ext/lzvn/FastCompression.h:8:10: fatal error: 'stdlib.h' file not found\n")', build.rs:34:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
  ```
  
  This issue seems to be caused by `bindgen` being unable to find the Windows include directories, as it seems to not be passed the proper arguments for directories as compared to the `cc` crate's builder. A temporary fix for this would be to include the proper directory from the cache of cargo-xwin using `.clang_args()', but that would not work cross-platform as each platform has a different location for the cache.

```
$ export BINDGEN_EXTRA_CLANG_ARGS_x86_64_pc_windows_msvc='--sysroot="/Users/plx/Library/Caches/cargo-xwin/xwin/sdk/"'
$ cargo xwin build --release --target x86_64-pc-windows-msvc
...
  exit status: 0
  cargo:rustc-link-lib=static=lzvn
  cargo:rustc-link-search=native=/Users/plx/Documents/Github/sepsplit-rs/target/x86_64-pc-windows-msvc/release/build/sepsplit-rs-hash/out
  cargo:rerun-if-changed=wrapper.h

  --- stderr
  ./src/ext/lzvn/FastCompression.h:8:10: fatal error: 'stdlib.h' file not found
  thread 'main' panicked at 'Unable to generate bindings: ClangDiagnostic("./src/ext/lzvn/FastCompression.h:8:10: fatal error: 'stdlib.h' file not found\n")', build.rs:35:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
it seems as though it can't find the proper headers even when passing in the SDK sysroot, would it perhaps require the include directories to be passed manually?

--- a/src/common.rs
+++ b/src/common.rs
@@ -160,6 +160,14 @@ impl XWinOptions {
                 cmd.env(format!("CFLAGS_{}", env_target), &cl_flags);
                 cmd.env(format!("CXXFLAGS_{}", env_target), &cl_flags);
 
+                cmd.env(
+                    format!("BINDGEN_EXTRA_CLANG_ARGS_{}", env_target), 
+                    format!(
+                        "-I{dir}/crt/include -I{dir}/sdk/include/ucrt -I{dir}/sdk/include/um -I{dir}/sdk/include/shared",
+                        dir = xwin_cache_dir.display()
+                    )
+                );
+
                 let target_arch = target
                     .split_once('-')
                     .map(|(x, _)| x)

